src/widget/wslidercomposed.cpp

```diff
@@ -141,7 +141,7 @@ void WSliderComposed::paintEvent(QPaintEvent *) {
     }
 
     if (!m_pHandle.isNull() && !m_pHandle->isNull()) {
-        double drawPos = m_handler.parameterToPosition(getControlParameterDisplay());
+        double drawPos = round(m_handler.parameterToPosition(getControlParameterDisplay()));
```

Comment:
We round() here, but we later use a QRectF, that allows painting on fractional pixel. 

What happens when we move to HiDPI support where we have to deal with screen and logic pixel. 

Do we have the case where it might be desired, like to reflect changes of amounts below 1 pixel resolution? 
The micro Shade key slider is such a candidate. But even without round, it is snapped to a pixel.